### PR TITLE
Merge intake overrcurrent branch

### LINKED ISSUE #94: Use the line break sensor and add a test button to dump the balls we're storing
Press button #11 on the operator controls to run the rollers in reverse.

### PR DESCRIPTION
**Pull request by maxh65000**
Resolves #94

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -43,6 +53,30 @@ public void setOutput(double output) {
     mRoller.set(ControlMode.PercentOutput, output);
   }
 
+  public boolean isBallDetected(){
```
Comment (ajwharton) [inline src/main/java/frc/robot/subsystems/Intake.java:60]: I cannot figure out what this method is trying to do with all of these counters. Max says that it's code that was already there, but it's not obvious. Comments would be great.

**Review by ajwharton — CHANGES_REQUESTED** [review]
I'd be ok with isBallDetected if somebody  could tell me what all of those counters do.

**Review by Team138Entropy — CHANGES_REQUESTED** [review]
Some kind of isBallDetected needs to be added back to dashboard periodic

File: src/main/java/frc/robot/subsystems/Intake.java (hunk 1 above)
Comment (maxh65000) [reply src/main/java/frc/robot/subsystems/Intake.java:60]: This was clarified with new comments

**Review by Team138Entropy — APPROVED** [review]
lack of isBallDetected explained, looks good
- liam

**Comment by maxh65000** [comment]
@ajwharton does this look good?

File: src/main/java/frc/robot/subsystems/Storage.java [hunk 2]
```diff
@@ -47,12 +47,12 @@ private Storage() {
   public void init() {
   }
 
-  public boolean isBallDetected() {
-    return (mIntakeSensor.get());
-  }
+  // public boolean isBallDetected() {
```
Comment (ajwharton) [inline src/main/java/frc/robot/subsystems/Storage.java]: Did you mean to comment this out

**Review by ajwharton — APPROVED** [review]
Approved as long as you meant to remove isBallDetected

File: src/main/java/frc/robot/subsystems/Storage.java (hunk 2 above)
Comment (maxh65000) [reply src/main/java/frc/robot/subsystems/Storage.java]: I removed this

**Comment by maxh65000** [comment]
We're not sure if we are going to stop the rollers when the line is broken or when the line is no longer broken. See [src/main/java/frc/robot/subsystems/Storage.java:60](https://github.com/Team138Entropy/Entropy2020/blob/IntakeOverrcurrent/src/main/java/frc/robot/subsystems/Storage.java#L57)

**Pull request merged**